src/bundle/Resources/views/content/tabs/translations.html.twig

```diff
@@ -1,30 +1,45 @@
 {% trans_default_domain "locationview" %}
 {{ form_start(actionsForm, {'action': path('ez_hybrid_platform_ui_translation_actions', {'contentId': content.id, 'redirectLocationId': location.id})}) }}
 
-<div class="ez-list-toolbar">
-    <h2 class="ez-list-toolbar-label">{{ 'locationview.translations.title'|trans|desc('Translation manager') }}</h2>
-    <div class="container-buttons">
-        {{ form_widget(actionsForm.delete, {'label': 'locationview.translations.delete.translations'|trans()|desc('Delete selected translations'), 'attr': {'class': 'ez-button ez-button-negative'}}) }}
-    </div>
-</div>
+{% set dialogId = 'confirm-translations-removal-' ~ location.id %}
+{{
+    include('@EzSystemsHybridPlatformUi/components/confirm_delete_dialog.html.twig', {
```

Comment:
We should considering replacing/completing this with a twig function. Something like:

```jinja2
{{ ez_confirm_dialog(
  dialogId=dialogId,
  message='confirm.remove.translations'|trans()|desc('Are you sure you want to remove selected translations?'),
  confirmButton: actionsForm.delete
) }}
```

It would ease backward compatibility (but could still use the template of course).

I'll try it here to validate that it works as expected.